[WIP] v6 Docs Dev (jschalz) (#2025)
loft-sh/devspace: docs/pages/configuration/images/build-settings/network.mdx

    image: john/appbackend
    network: host
```
**Explanation:**

> The image `backend` would be built using `docker` and `docker build` would be called using the `--network=host` flag.

Reviewer: ```suggestion
The image `backend` is built using `docker`, and `docker build` is called using the `--network=host` flag.
```
